s2a/src/main/java/io/grpc/s2a/handshaker/S2AStub.java

```diff
@@ -102,8 +102,7 @@ public SessionResp send(SessionReq req) throws IOException, InterruptedException
       if (exception != null) {
         throw new IOException(
             "Received an unexpected response from a host at the S2A's address. The S2A might be"
-                + " unavailable."
-                + exception.getMessage());
+                + " unavailable.",exception);
```

Comment:
Add a space after the comma. (Dunno why checkstyle didn't notice.)

Reply:
Done